<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>224</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="exitAddWindow">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="addToDB">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="clearEdit">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="nameAddLabel">
      <property name="text">
       <string>Name</string>
      </property>
      <property name="buddy">
       <cstring>lnameLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lnameLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="AliveAddcheckBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Alive</string>
   </property>
  </widget>
  <widget class="QLabel" name="diedAddLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>103</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Died</string>
   </property>
   <property name="buddy">
    <cstring>lnameLineEdit</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="diedAddSpinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>2010</number>
   </property>
   <property name="value">
    <number>1900</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="bornAddSpinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>71</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>2010</number>
   </property>
   <property name="value">
    <number>1900</number>
   </property>
  </widget>
  <widget class="QLabel" name="bornAddLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>71</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Born</string>
   </property>
   <property name="buddy">
    <cstring>lnameLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="genderAddLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>42</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
   <property name="buddy">
    <cstring>lnameLineEdit</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="gendrComboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <zorder>addToDB</zorder>
  <zorder>exitAddWindow</zorder>
  <zorder>clearEdit</zorder>
  <zorder>nameAddLabel</zorder>
  <zorder>lnameLineEdit</zorder>
  <zorder>lnameLineEdit</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
